### PR TITLE
basic-features.md: add the column for v8.0.0

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 1]
```diff
@@ -20,241 +20,241 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [DDL 算法断言](/sql-statements/sql-statement-alter-table.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| 在单条语句中添加多列 |  Y  |  Y  |  Y  | Y | Y | E | E | E | E | E | E | E |
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [临时表](/temporary-tables.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | N | N | N | N |
+| 并行 DDL |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [元数据锁](/metadata-lock.md) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL |  Y  | Y | Y | N | N | N | N | N | N | N | N | N |
+| [TiDB DDL V2 架构](/ddl-v2.md) | E | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:133]:
```suggestion
| [TiDB 加速建表](/fast-create-table.md) | E | E | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -20,241 +20,241 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | E | N | N | N | N | N | N | N | N | N |
+| [后台任务资源管控](/tidb-resource-control.md#管理后台任务) | E | E | E | N | N | N | N | N | N | N | N | N |
+| [TiFlash 存算分离架构与 S3 支持](/tiflash/tiflash-disaggregated-and-s3.md) | Y | Y | Y | E | N | N | N | N | N | N | N | N |
+| [选择执行分布式执行框架任务的 TiDB 节点](/system-variables.md#tidb_service_scope-从-v740-版本开始引入) | Y | Y | Y | N | N | N | N | N | N | N | N | N |
+| 通过系统变量 [`tidb_enable_tso_follower_proxy`](/system-variables.md#tidb_enable_tso_follower_proxy-从-v530-版本开始引入) 控制 PD Follower Proxy 功能 | Y | Y | Y | Y | Y | Y | Y | Y | N | N | N | N |
+| 通过系统变量 [`pd_enable_follower_handle_region`](/system-variables.md#pd_enable_follower_handle_region-从-v760-版本开始引入) 控制 [Active PD Follower](/tune-region-performance.md#通过-active-pd-follower-提升-pd-region-信息查询服务的扩展能力) 功能 | E | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:258]:
```suggestion
| [PD 微服务]((pd-microservices.md)) | Y | N | N | N | N | N | N | N | N | N | N | N |
```

Comment (qiancai) [reply basic-features.md:258]: @easonn7 PTAL

File: basic-features.md [hunk 3]
```diff
@@ -20,241 +20,241 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
 
 [^1]: TiDB 误将 latin1 处理为 utf8 的子集。见 [TiDB #18955](https://github.com/pingcap/tidb/issues/18955)。
```
Comment (Frank945946) [inline basic-features.md:260]:
```suggestion
| [TiDB 分布式执行框架](/system-variables#tidb_enable_dist_task-从-v710-版本开始引入) | Y | Y | Y | E | N | N | N | N | N | N | N | N |
| [全局排序](/system-variables#tidb_cloud_storage_uri-从-v740-版本开始引入) | Y | E | E | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -20,241 +20,241 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [DDL 算法断言](/sql-statements/sql-statement-alter-table.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| 在单条语句中添加多列 |  Y  |  Y  |  Y  | Y | Y | E | E | E | E | E | E | E |
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [临时表](/temporary-tables.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | N | N | N | N |
+| 并行 DDL |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [元数据锁](/metadata-lock.md) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL |  Y  | Y | Y | N | N | N | N | N | N | N | N | N |
+| [TiDB DDL V2 架构](/ddl-v2.md) | E | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (Frank945946) [inline basic-features.md:134]:
```suggestion
| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statement-admin-bdr-role#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N | N | N |
```

**Review by Frank945946 — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 5]
```diff
@@ -131,130 +131,130 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 | [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | N | N | N | N | N | N | N | N | N |
 | [TiDB 加速建表](/accelerated-table-creation.md) | N | N | N | N | N | N | N | N | N | N | N |
-
+| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statement-admin-bdr-role#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:134]:
```suggestion
| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 6]
```diff
@@ -131,130 +131,130 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 | [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | N | N | N | N | N | N | N | N | N |
 | [TiDB 加速建表](/accelerated-table-creation.md) | N | N | N | N | N | N | N | N | N | N | N |
-
+| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:134]:
```suggestion
| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statements/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 7]
```diff
@@ -131,130 +131,130 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 | [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | N | N | N | N | N | N | N | N | N |
 | [TiDB 加速建表](/accelerated-table-creation.md) | N | N | N | N | N | N | N | N | N | N | N |
-
+| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statements/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N | N | N |
 ## 事务
```
Comment (hfxsd) [inline basic-features.md:135]:
```suggestion

## 事务
```

File: basic-features.md [hunk 8]
```diff
@@ -131,130 +131,134 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 | [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | N | N | N | N | N | N | N | N | N |
 | [TiDB 加速建表](/accelerated-table-creation.md) | N | N | N | N | N | N | N | N | N | N | N |
+| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statements/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N | N | N |
 
 ## 事务
 
-| 事务 | 7.6 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+| 事务 | 8.0 | 7.6 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
 |---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
```
Comment (qiancai) [inline basic-features.md:139]:
```suggestion
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**